subsys/bluetooth/services/bas/Kconfig.bas

```diff
@@ -27,4 +27,9 @@ config BT_BAS_BLS_ADDITIONAL_STATUS_PRESENT
 	bool "Additional Battery Status Present"
 	help
 	Enable this option if Additional Battery Status information is present.
+
+config BT_BAS_BCS
+	bool "Battery Critical Status"
+	help
+	Enable this option to include Battery Critical Status Characteristic.
```

Comment:
Looks like this is an issue for this entire Kconfig file, however the help text should be indented with 1 tab + 2 spaces (maybe add a second prior commit to this PR to fix that for the other options in this file)

Reply:
Done Below PR fixes the indentation issue of BAS Kconfig
https://github.com/zephyrproject-rtos/zephyr/pull/79986

Reply:
Thanks. You could have included that as the first commit in this PR as well. Anyway, I marked the other PR as Trivial, so it should hopefully go in still today, and then you can rebase this one.